cli/bin/grain.js

```diff
@@ -42,9 +40,8 @@ function num(val) {
 }
 
 function graincVersion() {
-  const grainc = path.join(__dirname, "grainc.exe");
-
-  return execSync(`${grainc} --version`).toString().trim();
+  // TODO: This prints "unknown" for the JSOO version of grainc
```

Comment:
I think you said you wanted to just get rid of this? Probably fine if we're bundling the compiler in anyway.